src/classes/keyframe_scaler.py

```diff
@@ -0,0 +1,88 @@
+"""
+ @file
+ @brief This file listens to changes, and updates the primary project data
```

Comment:
Aaand, copy-pasting bites us again; forgot to change the file @brief from the original project_files source. Oops. I'll have to fix that.

(The list of authors, though, was intentional. I have no idea who wrote the `rescale_keyframes` function this is based on, so I decided the best approach was to just credit everyone listed in the original file.)

Reply:
Update: Fixed.